.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": ["es2015", "stage-0", "react"],
```

Comment:
Good point. I'll follow up with a PR to narrow down the list of transforms. We can probably get away with stage-3.